lib/improver/utilities/cube_metadata.py

```diff
@@ -145,11 +145,23 @@ def add_coord(cube, coord_name, changes, warnings_on=False):
         bounds = changes['bounds']
     units = None
     if 'units' in changes:
-        units = changes["units"]
-    new_coord = new_coord_method(long_name=coord_name,
-                                 points=points,
-                                 bounds=bounds,
-                                 units=units)
+        units = changes['units']
+    var_name = None
+    if 'var_name' in changes:
+        var_name = changes['var_name']
```

Comment:
Can you add something to the docstring to say keys are valid in `changes`?

Reply:
OK